src/table/table.scss

```diff
@@ -64,7 +64,6 @@
       transform: translateX(50%);
       touch-action: none;
       cursor: ew-resize;
-      z-index: 1;
```

Comment:
This needs to be reverted because the resizer needs to show up over the next column. Without this, the draggable area would become smaller.

Reply:
reverted